include/walker-dropdown.php

```diff
@@ -34,11 +34,12 @@ class PLL_Walker_Dropdown extends PLL_Walker {
 	public function start_el( &$output, $element, $depth = 0, $args = array(), $current_object_id = 0 ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
 		$value_type = $args['value'];
 		$output .= sprintf(
-			"\t" . '<option value="%1$s"%2$s%3$s>%4$s</option>' . "\n",
+			"\t" . '<option value="%1$s"%2$s%3$s data-pll-lang="%5$s">%4$s</option>' . "\n",
```

Comment:
I don't believer we need a prefix here:
```suggestion
			"\t" . '<option value="%1$s"%2$s%3$s data-lang="%5$s">%4$s</option>' . "\n",
```